docs-release-notes.md

```diff
@@ -1,10 +1,18 @@
+# June 16, 2023
+
+## Device Sync
+
+- App Builder/Tiered Device Sync: Clarify the opening Tip about starting a PoC with a Template App to get an App pre-configured for Device Sync
```

Comment:
Having trouble finding where this is in the TOC (don't see a 'Tiered Device Sync' page under App Builder's Resources)

Reply:
Yeah, this is (intentionally) a private page currently, so it does not appear in the ToC. Well spotted, though!